site/layouts/partials/docs-sidebar.html

```diff
@@ -16,16 +16,16 @@
     {{- $active_group := eq $.Page.Params.group $group_slug }}
 
     <li class="bd-sidenav-group my-1{{ if $active_group }} active{{ end }} js-sidenav-group{{ if $group.pages }} has-children{{ end }}">
-      <a class="d-inline-flex align-items-center bd-sidenav-group-link" href="/docs/{{ $.Site.Params.docs_version }}/{{ $group_slug }}/{{ if $group.pages }}{{ $link_slug }}/{{ end }}">
-        <div>{{ $group.title }}</div>
```

Comment:
I remember I saw this before and I had removed but the patch probably never landed. I can't find any reason to keep the `div` here.

Reply:
Neither do I, I searched across scripts and styles, at every breakpoints, and didn't find any usage of this `div`…